README.md

```diff
@@ -4,12 +4,12 @@ checkers
  [![Build Status](https://github.com/mackerelio/checkers/workflows/Build/badge.svg?branch=master)][actions]
 [![Coverage Status](https://coveralls.io/repos/mackerelio/checkers/badge.svg?branch=master)][coveralls]
 [![Apache License Version 2.0](http://img.shields.io/badge/license-APACHE2-blue.svg)][license]
```

Comment:
This badge link can use https.

Reply:
I did! [f9f921d](https://github.com/mackerelio/checkers/pull/16/commits/f9f921dde8485b7a2cf34732754ebfdc39091409)